test/lift.js

```diff
@@ -40,4 +41,10 @@ describe('lift', function() {
     it('throws on zero arguments', function() {
         assert.throws(R.lift);
     });
+
+    it('works with other functors such as "Maybe"', function() {
+        var addM = R.lift(R.add);
+        assert.deepEqual(addM(Maybe(3), Maybe(5)), Maybe(8));
+    });
```

Comment:
I'd rather use a simpler type here. Something like [Id][1] but with only the required methods implemented.


[1]: https://github.com/fantasyland/fantasy-land/blob/master/id.js